Services/Masterdata/Services.Masterdata.UnitTests/MasterdataServiceUnitTests.cs

```diff
@@ -11,6 +11,7 @@ public MasterdataServiceUnitTests(MasterdataServiceFixture fixture)
         {
             _fixture = fixture;
         }
+
         [Theory]
         [InlineData("123", "Name", "Description", "{ \"someKey\": \"some value\" }", "123", "Name", "Description", "{ \"someKey\": \"some value\" }")]
         [InlineData("123", "Name", "This is bad: <script>alert('xss')</script>, should be sanitized", "{ \"someKey\": \"some value\" }", "123", "Name", "This is bad: , should be sanitized", "{ \"someKey\": \"some value\" }")]
```

Comment:
I don't think we should remove the sanitize tests. What is the reason that these are removed?

Reply:
> I don't think we should remove the sanitize tests. What is the reason that these are removed?

please give me more details?

Reply:
We didn't remove them, just moved them (using an extension method)

Reply:
You're right, my bad. I'll resolve this conversation